[2022-08-29T12:35:17.103-0400] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop scheduled__2022-08-28T00:00:00+00:00 [queued]>
[2022-08-29T12:35:17.118-0400] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop scheduled__2022-08-28T00:00:00+00:00 [queued]>
[2022-08-29T12:35:17.118-0400] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-08-29T12:35:17.118-0400] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2022-08-29T12:35:17.118-0400] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-08-29T12:35:17.162-0400] {taskinstance.py:1389} INFO - Executing <Task(BashOperator): run_after_loop> on 2022-08-28 00:00:00+00:00
[2022-08-29T12:35:17.166-0400] {standard_task_runner.py:52} INFO - Started process 537 to run task
[2022-08-29T12:35:17.169-0400] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'run_after_loop', 'scheduled__2022-08-28T00:00:00+00:00', '--job-id', '7', '--raw', '--subdir', '/home/augenz/PycharmProjects/airflow/.env_airflow/lib/python3.9/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/run/user/1000/app/com.jetbrains.PyCharm-Community/tmpkje6ucro', '--error-file', '/run/user/1000/app/com.jetbrains.PyCharm-Community/tmp76ak2283']
[2022-08-29T12:35:17.170-0400] {standard_task_runner.py:80} INFO - Job 7: Subtask run_after_loop
[2022-08-29T12:35:17.342-0400] {task_command.py:371} INFO - Running <TaskInstance: example_bash_operator.run_after_loop scheduled__2022-08-28T00:00:00+00:00 [running]> on host pop-os.localdomain
[2022-08-29T12:35:17.533-0400] {taskinstance.py:1581} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=run_after_loop
AIRFLOW_CTX_EXECUTION_DATE=2022-08-28T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-28T00:00:00+00:00
[2022-08-29T12:35:17.534-0400] {subprocess.py:62} INFO - Tmp dir root location: 
 /run/user/1000/app/com.jetbrains.PyCharm-Community
[2022-08-29T12:35:17.535-0400] {subprocess.py:74} INFO - Running command: ['/usr/bin/bash', '-c', 'echo 1']
[2022-08-29T12:35:17.543-0400] {subprocess.py:85} INFO - Output:
[2022-08-29T12:35:17.546-0400] {subprocess.py:92} INFO - 1
[2022-08-29T12:35:17.547-0400] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-29T12:35:17.642-0400] {taskinstance.py:1407} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=run_after_loop, execution_date=20220828T000000, start_date=20220829T163517, end_date=20220829T163517
[2022-08-29T12:35:17.707-0400] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-29T12:35:17.811-0400] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
